### PR TITLE
*: Support time

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: mysql/time.go [hunk 1]
```diff
@@ -395,7 +395,7 @@ func parseDatetime(str string, fsp int) (Time, error) {
 	)
 
 	seps := parseDateFormat(str)
-
+	fmt.Println("seps len:", len(seps))
```
Comment (shenli) [inline mysql/time.go]: Remove this line?

File: evaluator/builtin_time.go [hunk 2]
```diff
@@ -395,6 +395,29 @@ func builtinCurrentTime(args []types.Datum, _ context.Context) (d types.Datum, e
...
+func builtinTime(args []types.Datum, _ context.Context) (d types.Datum, err error) {
+	if args[0].Kind() == types.KindNull {
+		return
+	}
+
+	str, err := args[0].ToString()
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	idx := strings.Index(str, ".")
```
Comment (shenli) [inline evaluator/builtin_time.go]: How about more than one dot?

**Comment by shenli** [comment]
LGTM

File: evaluator/builtin_time.go [hunk 3]
```diff
@@ -395,6 +395,29 @@ func builtinCurrentTime(args []types.Datum, _ context.Context) (d types.Datum, e
...
+	fsp := 0
+	if idx != -1 && idx != 0 {
```
Comment (coocood) [inline evaluator/builtin_time.go]: `idx != 0` seems not needed.

File: evaluator/builtin_time.go (hunk 2 above)
Comment (coocood) [inline evaluator/builtin_time.go]: MySQL will truncate it to "10:10:10.123".

**Comment by coocood** [comment]
LGTM

**Pull request merged**